### PR TITLE
Sameer/n tracks

### PR DESCRIPTION
**Pull request by crazzywizard**
### Purpose
Increase ability to remix more than 2 tracks

* added n number of tracks to mix
* added ability to preview sounds
* loading staked tracks

### CODE REVIEW

File: components/GameScreen/GameScreen.tsx [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
+  description: string
+  properties: {
+    name: string
+    number: number
+  }
+  tokenId: number
+}
+
+const getProvider = () => {
+  const goerliRpc = "https://ethereum-goerli-rpc.allthatnode.com"
```
Comment (sweetmantech) [inline components/GameScreen/GameScreen.tsx:12]: this should be ENV variable.

I will update in my PR, feel free to merge this and I will update 👍

File: components/GameScreen/GameScreen.tsx [hunk 2]
```diff
@@ -24,76 +48,186 @@ const GameScreen = ({ onSuccess }: any) => {
...
+      newOptions.push({
+        id: uri.name,
+        name: uri.name,
+        imgUrl: getIpfsLink(uri.image),
+        musicUrl: getIpfsLink(uri.animation_url),
+      })
+    })
+    setOptions([...options, ...newOptions])
+    setLoadingAssets(false)
+  }, [options])
```
Comment (sweetmantech) [inline components/GameScreen/GameScreen.tsx:61]: nit: we should abstract this to a helper function.

File: components/GameScreen/GameScreen.tsx [hunk 3]
```diff
@@ -24,76 +48,186 @@ const GameScreen = ({ onSuccess }: any) => {
...
-                <source src={musicUrl} type="audio/mpeg" />
-                <track kind="captions" />
-              </audio>
-            )}
-          </div>
-        ))}
+          ))}
+        {choices.length > 1 && (
+          <MintButton onSuccess={onSuccess} audioTracksToMix={chosenAudioTracks} />
+        )}
```
Comment (sweetmantech) [inline components/GameScreen/GameScreen.tsx:160]: this component is starting to get rather larger, any chance we can break this out into smaller children?

File: lib/purchase.tsx [hunk 4]
```diff
@@ -8,7 +8,7 @@ const purchase = async (signer: Signer, initialData?: string) => {
 
   const contract = new ethers.Contract(contractAddress, abi, signer)
   try {
-    const tx = await contract.purchase(1, initialData)
+    const tx = await contract.purchase(1, initialData, { value: ethers.utils.parseEther("0.001") })
```
Comment (sweetmantech) [inline lib/purchase.tsx:11]: we'll want this to be read from the contract eventually but this is cool for now 👍

File: next.config.js [hunk 5]
```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  images: {
+    domains: ["images.unsplash.com", "nftstorage.link"],
```
Comment (sweetmantech) [inline next.config.js]: what are we using `unsplash` for? 

any reason to not store those photos on nft storage?

File: pages/api/remix/index.ts [hunk 6]
```diff
@@ -14,12 +15,14 @@ const ffmpeg = require("fluent-ffmpeg")
 
 ffmpeg.setFfmpegPath(ffmpegStatic)
 
-const mixAudio = (track1: string, track2: string, output: string) =>
+const mixAudio = (tracks: string[], output: string) =>
```
Comment (sweetmantech) [inline pages/api/remix/index.ts:18]: 🔥 🔥 🔥

File: pages/index.tsx [hunk 7]
```diff
@@ -7,7 +7,7 @@ import LaunchPage from "../components/LaunchPage"
 import Results from "../components/Results"
 import Chat from "../components/Chat"
 
-const GameScreen = dynamic(() => import("../components/GameScreen"))
+const GameScreen = dynamic(() => import("../components/GameScreen"), { ssr: false })
```
Comment (sweetmantech) [inline pages/index.tsx]: why are we resetting `ssr: false`? 

is this an intentional change?

**Review by sweetmantech — APPROVED** [review]
ship it 🚢

File: next.config.js (hunk 5 above)
Comment (crazzywizard) [reply next.config.js]: not using it for anything

File: pages/index.tsx (hunk 7 above)
Comment (crazzywizard) [reply pages/index.tsx]: that was my bad, this was when I was debugging for audio playback and switched to crunker

**Pull request merged**